myhomework5.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+require_once 'HttpStatusCodes.php';
+const HOST_BASE_DIRECTORIES = [
```

Comment:
don't see these folders in PR

Reply:
I didn't think these folders should be put on the gitHub